### PR TITLE
CSI 3191 checking not available in XIV

### PR DESCRIPTION
**Pull request by zingero**

### CODE REVIEW

File: controller/array_action/array_mediator_xiv.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 LUN_IS_ALREADY_IN_USE_ERROR = "LUN is already in use"
 UNDEFINED_MAPPING_ERROR = "The requested mapping is not defined"
 NO_ALLOCATION_SPACE_ERROR = "No space to allocate to the volume"
+NOT_AVAILABLE = "Not Available"
```
Comment (ArbelNathan) [inline controller/array_action/array_mediator_xiv.py:22]: I prefer if we could be not case sensitive..

Comment (zingero) [reply controller/array_action/array_mediator_xiv.py:22]: What do you mean? This is the value we get from the storage.

Comment (oriyarde) [reply controller/array_action/array_mediator_xiv.py:22]: I assume he meant to compare both to normalized values (e.g. `actual_value.lower() == NOT_AVAILABLE.lower()`)

Comment (oriyarde) [reply controller/array_action/array_mediator_xiv.py:22]: not sure if it is better

**Review by oriyarde — APPROVED** [review]
LGTM

File: controller/array_action/array_mediator_xiv.py (hunk 1 above)
Comment (ArbelNathan) [reply controller/array_action/array_mediator_xiv.py:22]: we are doing it with all error codes but if we think it's not necessary we can leave it

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**